buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -80,6 +98,26 @@
       }
     },
     {
+        "enabled": true,
+        "continueOnError": false,
+        "alwaysRun": false,
+        "displayName": "Inject signed symbol catalogs",
+        "timeoutInMinutes": 0,
+        "task": {
+            "id": "e213ff0f-5d5c-4791-802d-52ea3e7be1f1",
+            "versionSpec": "1.*",
+            "definitionType": "task"
+        },
+        "inputs": {
+            "scriptType": "inlineScript",
+            "scriptName": "",
+            "arguments": "-ConfigGroup $(ConfigurationGroup) -SymPkgGlob $(AzureContainerSymbolPackageGlob) -PipelineSrcDir $(Pipeline.SourcesDirectory)",
+            "workingFolder": "$(Pipeline.SourcesDirectory)",
+            "inlineScript": "param($ConfigGroup, $SymPkgGlob, $PipelineSrcDir)\nmsbuild build.proj /t:InjectSignedSymbolCatalogIntoSymbolPackages /p:SymbolPackagesToPublishGlob=$PipelineSrcDir\\packages\\AzureTransfer\\$ConfigGroup\\$env:AzureContainerSymbolPackageGlob /p:SymbolCatalogCertificateId=400",
```

Comment:
Does this mean that the packages we have in Azure blob storage are no the same as what we publish to myget?